packages/storage/index.ts

```diff
@@ -17,7 +17,10 @@
 
 import firebase from '@firebase/app';
 import { FirebaseApp } from '@firebase/app-types';
-import { FirebaseServiceFactory } from '@firebase/app-types/private';
+import {
+  FirebaseServiceFactory,
+  _FirebaseNamespace
+} from '@firebase/app-types/private';
```

Comment:
Could this just be `FirebaseNamespace`? It's not immediately obvious to me why it needs an underscore, but I am willing to learn :)

Reply:
Hm, I guess we have:

```
import { FirebaseNamespace } from '@firebase/app-types';
import { _FirebaseNamespace } from '@firebase/app-types/private';	
```

Reply:
Sure, the public `FirebaseNamespace` type doesn't have an `INTERNAL` property which causes a TS error on line 54.